### PR TITLE
Autocomplete: Change demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

**Review by AnnaKondratova — DISMISSED** [review]

File: JSDemos/Demos/Autocomplete/Overview/description.md [hunk 1]
```diff
@@ -1 +1,42 @@
-The Autocomplete component is&nbsp;a&nbsp;textbox that provides suggestions while a&nbsp;user types into&nbsp;it.
\ No newline at end of file
+Autocomplete is a text box that displays suggestions while a user types into it. This demo shows how you can customize the Autocomplete component and how to handle its value change.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Autocomplete/Overview/description.md]:
```suggestion
Autocomplete is a text box that displays suggestions while a user types. This demo shows how to customize the Autocomplete component and handle value changes.
```

File: JSDemos/Demos/Autocomplete/Overview/description.md [hunk 2]
```diff
@@ -1 +1,42 @@
...
+
+## Bind Autocomplete to Data
+
+Use one of the following properties to supply data for the component:
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Autocomplete/Overview/description.md]:
```suggestion
Use one of the following properties to supply data to the component:
```

File: JSDemos/Demos/Autocomplete/Overview/description.md [hunk 3]
```diff
@@ -1 +1,42 @@
...
+
+* [items[]](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/)             
+Accepts a local data array.
+
+* [dataSource](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#dataSource)            
+Accepts a local data array, a [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object, or one of DevExtreme data stores. In this demo, every Autocomplete component is bound to a local array, except the "Custom Item Template and Data Source Usage" component that uses an [ODataStore](/Documentation/ApiReference/Data_Layer/ODataStore/).
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Autocomplete/Overview/description.md]:
```suggestion
Accepts a local data array, [DataSource](/Documentation/ApiReference/Data_Layer/DataSource/) object, or DevExtreme data store. In this demo, every Autocomplete component is bound to a local array, except the "Custom Item Template and Data Source Usage" component that uses an [ODataStore](/Documentation/ApiReference/Data_Layer/ODataStore/).
```

File: JSDemos/Demos/Autocomplete/Overview/description.md [hunk 4]
```diff
@@ -1 +1,42 @@
...
+
+The [value](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#value) property stores the selected item. You can use the same property to select an item programmatically, as shown in the "Disabled" Autocomplete component.
+
+If the data source contains objects, you should set the [valueExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#valueExpr) property. It accepts a data field name that uniquely identifies each data object. In this demo, **valueExpr** is set only for the "Custom Item Template and Data Source Usage" Autocomplete because others are bound to arrays of primitive values.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Autocomplete/Overview/description.md]: `you should set the [valueExpr] property` Set it to what?

File: JSDemos/Demos/Autocomplete/Overview/description.md [hunk 5]
```diff
@@ -1 +1,42 @@
...
+
+## Configure Search Parameters
+
+When an end user types the first character, Autocomplete displays suggestions. To increase the number of characters that triggers suggestions, use the [minSearchLength](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#minSearchLength) property. You can also specify the time interval Autocomplete should wait before it displays suggestions. Assign this time interval in milliseconds to the [searchTimeout](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#searchTimeout) property. See the "With Custom Search Options" Autocomplete component for an example.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Autocomplete/Overview/description.md]:
```suggestion
When a user types the first character, Autocomplete displays suggestions. To increase the number of characters that triggers suggestions, use the [minSearchLength](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#minSearchLength) property. You can also specify the time interval Autocomplete should wait before it displays suggestions. Assign this time interval in milliseconds to the [searchTimeout](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#searchTimeout) property. See the "With Custom Search Options" Autocomplete component for an example.
```

File: JSDemos/Demos/Autocomplete/Overview/description.md [hunk 6]
```diff
@@ -1 +1,42 @@
...
+
+Usually, the data field that supplies Autocomplete with suggestions (**valueExpr**) is the same data field that is searched for the typed text. If in your case they are two different fields, assign the field that supplies Autocomplete with suggestions to the **valueExpr** property and the field to be searched - to the [searchExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#searchExpr) property. Note that **searchExpr** also accepts arrays in case you need several fields to search in.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Autocomplete/Overview/description.md]:
```suggestion
Usually, the data field that supplies Autocomplete with suggestions (**valueExpr**) is the same data field that is searched for the typed text. If you use two different fields, assign the field that supplies Autocomplete with suggestions to the **valueExpr** property and the field to be searched to the [searchExpr](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#searchExpr) property. Note that **searchExpr** also accepts arrays if you want to search multiple fields.
```

File: JSDemos/Demos/Autocomplete/Overview/description.md [hunk 7]
```diff
@@ -1 +1,42 @@
...
+
+## Handle Value Change
+
+To handle value changes, implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#onValueChanged) function. In this demo, we use **onValueChanged** to display values of all Autocomplete components.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Autocomplete/Overview/description.md]:
```suggestion
To handle value changes, implement the [onValueChanged](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#onValueChanged) function. In this demo, we use **onValueChanged** to display the values of all Autocomplete components.
```

File: JSDemos/Demos/Autocomplete/Overview/description.md [hunk 8]
```diff
@@ -1 +1,42 @@
...
+
+## Appearance Customization
+
+To customize the Autocomplete appearance, you can specify the following properties:
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Autocomplete/Overview/description.md]:
```suggestion
You can specify the following properties to customize the Autocomplete component's appearance:
```

File: JSDemos/Demos/Autocomplete/Overview/description.md [hunk 9]
```diff
@@ -1 +1,42 @@
...
+
+* [placeholder](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#placeholder)             
+Sets a text string displayed when Autocomplete is empty.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Autocomplete/Overview/description.md]:
```suggestion
The text that is displayed when Autocomplete is empty. 
```

**Review by dirk-pieterse — DISMISSED** [review]

File: JSDemos/Demos/Autocomplete/Overview/description.md (hunk 4 above)
Comment (vladaskorohodova) [reply JSDemos/Demos/Autocomplete/Overview/description.md]: I'll replace "set" with "specify"

**Review by AnnaKondratova — APPROVED** [review]

**Pull request merged**